src/vm/checker/typecheck/contexts.rs

```diff
@@ -24,6 +24,8 @@ pub struct ContractAnalysis {
     read_only_function_types: BTreeMap<String, FunctionType>,
     map_types: BTreeMap<String, (TypeSignature, TypeSignature)>,
     persisted_variable_types: BTreeMap<String, TypeSignature>,
+    tokens: BTreeSet<String>,
+    assets: BTreeMap<String, TypeSignature>,
```

Comment:
should we also update the references to `tokens` and `assets` in the code? 👹

Reply:
I'll update in the next PR (which will be refactoring a bunch of the vm code to support the more general K-V interface of MARFs)